decapode/kafka/consumer.py

```diff
@@ -0,0 +1,54 @@
+import json
+from kafka import KafkaConsumer
+import logging
+
+from decapode import context
+from decapode.config import KAFKA_URI, KAFKA_API_VERSION
+
+log = logging.getLogger("decapode-kafka")
+
+
+async def consume_kafka_datasets() -> None:
+    '''For each message received in Kafka topic dataset, set all resources of the dataset to
+    priority for crawling.
+    '''
+    kafka_api_version = tuple([int(value) for value in KAFKA_API_VERSION.split('.')])
+    consumer = KafkaConsumer(
+        'resource.created',
+        'resource.modified',
+        'resource.deleted',
+        bootstrap_servers=KAFKA_URI,
+        group_id=None,
+        api_version=kafka_api_version,
+        reconnect_backoff_max_ms=100000,  # TODO: what value to set here?
+        # auto_offset_reset='smallest' # This line can be added for tests to receive messages from the beginning
+    )
+
+    log.info('Consuming resource.created & resource.modified topics...')
+    for message in consumer:
```

Comment:
Should we check for `service` in message? Attaching only to messages produced by `udata` for example.

Reply:
If Decapode starts consuming and producing resource.modified messages yes. Otherwise it is not necessary for now